<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataChart</class>
 <widget class="QWidget" name="DataChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="hTickCountLabel">
     <property name="text">
      <string>H tick count</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="exportButton">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QSpinBox" name="vMinorTickCountSpinBox"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="vTickCountLabel">
     <property name="text">
      <string>V tick count</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="hTickCountSpinBox">
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="hMinorTickCountSpinBox"/>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="vTickCountSpinBox">
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="logFitButton">
     <property name="text">
      <string>Log fit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
